lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -230,17 +230,30 @@ def __init__(self, ufeSceneItem):
         # Get the UFE Attributes interface for this scene item.
         self.attrS = ufe.Attributes.attributes(self.item)
 
+        self.showArrayAttributes = cmds.optionVar(query="mayaUSD_AEShowArrayAttributes")
+
         cmds.editorTemplate(beginScrollLayout=True)
         self.buildUI()
         cmds.editorTemplate(addExtraControls=True)
         self.createMetadataSection()
         cmds.editorTemplate(endScrollLayout=True)
 
+        # Suppress all array attributes not used
+        if not self.showArrayAttributes:
+            for attrName in self.attrS.attributeNames:
+                if self.isArrayAttribute(attrName):
+                    self.suppress(attrName)
```

Comment:
Maya code will issue a warning if you "add then suppress" or "suppress and then add" an attribute. You are only supposed to call one or the other. So you should rework this somehow to avoid calling the addControls() on an item that you suppressed here.